cogs/mod.py

```diff
@@ -36,28 +36,28 @@ def __init__(self, bot):
     async def on_ready(self):
         self.guild = self.bot.get_guild(self.bot.config.server_id)
         self.verification_channel = utils.get(self.guild.text_channels, name=self.bot.config.verification_channel)
+        if self.verification_channel:
```

Comment:
it'd be better to do an early return here

Reply:
resolved in e6c43e4